test/integration/externalentity_test.go

```diff
@@ -184,8 +184,11 @@ var _ = Describe(fmt.Sprintf("%s,%s: ExternalEntity", focusAws, focusAzure), fun
 	}
 	table.DescribeTable("Test ExternalEntity Life cycle",
 		func(kind string, hasNic, hasPort bool) {
+			// TODO: VM CRD is no longer present
```

Comment:
@shenmo3 can you update the test similar to stale member test. Upon Ces and check if VM/EE is removed